slasher/db/iface/interface.go

```diff
@@ -31,7 +31,7 @@ type ReadOnlyDatabase interface {
 	LatestIndexedAttestationsTargetEpoch(ctx context.Context) (uint64, error)
 
 	// MinMaxSpan related methods.
-	EpochSpans(ctx context.Context, epoch uint64) (*detectionTypes.EpochStore, error)
+	EpochSpans(ctx context.Context, epoch uint64, fromCache bool) (*detectionTypes.EpochStore, error)
```

Comment:
we can check in cache as default

Reply:
I thought about this, golang lacks default parameters, and I think it would be weird to have a DB function default to using cache. Wdyt?

Reply:
if one method store in cache the other one should benefit from it regardless if it knew its going to be there